### DISCUSSION TITLE
Add multiple languages support for the dashboard interface

### DISCUSSION THREAD

**Discussion by Recreabox** [dailydotdev/daily#1283] [Feature Requests]
Hello !

First, thank for your daily.dev browser extension. It's very useful and have very good features and UI.

I know all the news are in english and that's ok.
But i wanted to ask if you can add the feature to choose the dashboard interface language ?

Thank you for your work

**Comment by idoshamun** [comment]
Hi @Recreabox,

Thanks a lot for the suggestion. We will consider it 🙏
